packages/react/src/components/UIShell/components/SideNav.tsx

```diff
@@ -599,6 +602,12 @@ SideNav.propTypes = {
    */
   expanded: PropTypes.bool,
 
+  /**
+   * Specify the breakpoint at which the SideNav will be hidden.
+   * Can be one of `sm`, `md`, `lg`, `xlg`, or `max`.
+   */
+  hideOnBreakpoint: PropTypes.oneOf(['sm', 'md', 'lg', 'xlg', 'max']),
```

Comment:
We should probably add an additional comment line

`* Only applies when `isRail` is true.`